projects/schematics/src/shared/constants.ts

```diff
@@ -218,6 +219,9 @@ export const ANONYMOUS_CONSENT_LAUNCH_DIALOG_SERVICE =
   'AnonymousConsentLaunchDialogService';
 export const ANONYMOUS_CONSENT_OPEN_DIALOG_COMPONENT =
   'AnonymousConsentOpenDialogComponent';
+
+export const TEST_OUTLET_MODULE = 'TestOutletModule';
```

Comment:
Where this is used?

Reply:
This one is used in dev-schematics. We simply didn't want to create separate constants.ts file for dev-schematics so we decided to put it in here and include in public api.